packages/mediacenter/TexturePacker/package.mk

```diff
@@ -8,15 +8,14 @@ PKG_LICENSE="GPL"
 PKG_SITE="http://www.kodi.tv"
 PKG_DEPENDS_HOST="lzo:host libpng:host libjpeg-turbo:host giflib:host"
 PKG_DEPENDS_UNPACK="kodi"
-PKG_NEED_UNPACK="$(get_pkg_directory $MEDIACENTER)"
 PKG_LONGDESC="kodi-platform:"
 
-PKG_CMAKE_SCRIPT="$(get_build_dir $MEDIACENTER)/tools/depends/native/TexturePacker/CMakeLists.txt"
+PKG_CMAKE_SCRIPT="$(get_build_dir kodi)/tools/depends/native/TexturePacker/CMakeLists.txt"
 
 PKG_CMAKE_OPTS_HOST="-Wno-dev"
 
 pre_configure_host() {
-  export CXXFLAGS="$CXXFLAGS -std=c++11 -DTARGET_POSIX -DTARGET_LINUX -D_LINUX -I$(get_build_dir $MEDIACENTER)/xbmc/platform/linux"
+  export CXXFLAGS="$CXXFLAGS -std=c++11 -DTARGET_POSIX -DTARGET_LINUX -D_LINUX -I$(get_build_dir kodi)/xbmc/platform/linux"
```

Comment:
Same as above, please use `$MEDIACENTER` instead of `kodi`, and `PKG_DEPENDS_UNPACK="kodi"` should be changed to `PKG_DEPENDS_UNPACK="$MEDIACENTER"`

Reply:
done